### PR TITLE
Update gist example in the docs.

### PR DESCRIPTION
**Pull request by pranshu-taneja**
Updated both the links as per their tasks. Now working perfectly.

**Load a GIST and have it be visible in the Editor** https://remix.ethereum.org/?#activate=solidity,fileManager&gist=0fe90e825327ef313c88aedfe66ec142&call=fileManager//open//gist-0fe90e825327ef313c88aedfe66ec142/gridMix4.sol 

**Load a GIST, have it be visible in the Editor & load a list of plugins** https://remix.ethereum.org/?#activate=solidity,pluginManager&gist=0fe90e825327ef313c88aedfe66ec142&call=fileManager//open//gist-0fe90e825327ef313c88aedfe66ec142/gridMix4.sol

### CODE REVIEW

**Comment by pranshu-taneja** [comment]
Here is the issue link posted on remix project GitHub repository:
https://github.com/ethereum/remix-project/issues/951

File: docs/locations.md [hunk 1]
```diff
@@ -156,12 +156,12 @@ https://remix.ethereum.org/?gist=0fe90e825327ef313c88aedfe66ec142
...
 Using both `gist` & `call`
 ```
-https://remix.ethereum.org/?#activate=solidity,debugger&gist=0fe90e825327ef313c88aedfe66ec142&call=fileManager//open//browser/gists/0fe90e825327ef313c88aedfe66ec142/gridMix4.sol
+https://remix.ethereum.org/?#activate=solidity,fileManager&gist=0fe90e825327ef313c88aedfe66ec142&call=fileManager//open//gist-0fe90e825327ef313c88aedfe66ec142/gridMix4.sol 
 ```
 
 ### Load a GIST, have it be visible in the Editor & load a list of plugins:
 ```
-https://remix.ethereum.org/?#activate=solidity,LearnEth&gist=0fe90e825327ef313c88aedfe66ec142&call=fileManager//open//browser/gists/0fe90e825327ef313c88aedfe66ec142/gridMix4.sol 
+https://remix.ethereum.org/?#activate=solidity,pluginManager&gist=0fe90e825327ef313c88aedfe66ec142&call=fileManager//open//gist-0fe90e825327ef313c88aedfe66ec142/gridMix4.sol
```
Comment (Aniket-Engg) [inline docs/locations.md:164]: `pluginManager ` is loaded by default. You can revert this change

**Review by Aniket-Engg — CHANGES_REQUESTED** [review]

**Review by pranshu-taneja — COMMENTED** [review]
The change is on **Line 164** not on **Line 159** (mistakenly typed)

**Review by Aniket-Engg — APPROVED** [review]

**Comment by Aniket-Engg** [comment]
Thanks @pranshu-taneja !!!

**Pull request merged**